integration_tests/e2e/reviewPlan/exemption/confirmExemption.cy.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Cypress tests that test the Confirm Review Exemption page
+ */
+
+import ReviewPlanExemptionReasonValue from '../../../../server/enums/reviewPlanExemptionReasonValue'
+import OverviewPage from '../../../pages/overview/OverviewPage'
+import Page from '../../../pages/page'
+import ConfirmExemptionPage from '../../../pages/reviewPlan/exemption/confirmExemptionPage'
+import ExemptionReasonPage from '../../../pages/reviewPlan/exemption/exemptionPage'
+import ExemptionRecordedPage from '../../../pages/reviewPlan/exemption/exemptionRecordedPage'
+
+context(`Confirm review exemption page`, () => {
+  const prisonNumber = 'G6115VJ'
+  beforeEach(() => {
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.signIn()
+    cy.visit(`/plan/${prisonNumber}/review/exemption`)
+  })
+
+  it(`Should navigate to the 'Overview' page when 'No, go back to learning and work progress plan' button on Confirm review exemption page is clicked`, () => {
+    // Given
+    Page.verifyOnPage(ExemptionReasonPage) //
+      .selectExemptionReason(ReviewPlanExemptionReasonValue.EXEMPT_PRISONER_DRUG_OR_ALCOHOL_DEPENDENCY)
+      .enterExemptionReasonDetails(
+        ReviewPlanExemptionReasonValue.EXEMPT_PRISONER_DRUG_OR_ALCOHOL_DEPENDENCY,
+        'In treatment',
+      )
+      .submitPage()
+
+    // When
+    const confirmExemptionPage = Page.verifyOnPage(ConfirmExemptionPage)
+
+    // Then
+    confirmExemptionPage.goBackToLearningAndWorkProgressPlan()
+    Page.verifyOnPage(OverviewPage)
+  })
```

Comment:
Minor improvement suggestion .... do you think it is worth asserting the success message banner is not displayed? I think we have a suitable method already on `OverviewPage`, so I thnk you can do:
```
    Page.verifyOnPage(OverviewPage) //
      .doesNotHaveSuccessMessage()
```